src/Domain/StagerInterface.php

```diff
@@ -2,34 +2,51 @@
 
 namespace PhpTuf\ComposerStager\Domain;
 
+/**
+ * Executes a Composer command in the staging directory.
```

Comment:
These changes make sense but I don't understand why they're in scope for this PR. That said, not a big deal.

Reply:
Yeah, they came along with the other added documentation of my interfaces you suggested. :)